<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
   SPDX-FileCopyrightText: 2020 Bill Binder &lt;dxtwjb@gmail.com&gt;
   SPDX-License-Identifier: GPL-2.0-only
 </comment>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vizzyix - A Portage Package Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="categoryTree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
       </property>
      </widget>
      <widget class="QTableView" name="packageListView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="packageDescription">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableView" name="ebuildList">
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectAll"/>
   <addaction name="actionSelectInstalled"/>
   <addaction name="actionSelectWorld"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Reload EIX database</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/pkgtable/images/all-marker.png</normaloff>:/pkgtable/images/all-marker.png</iconset>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="toolTip">
    <string>Show all packages</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/app/images/eyeball.png</normaloff>:/app/images/eyeball.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Vizzyix ...</string>
   </property>
  </action>
  <action name="actionSelectWorld">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/pkgtable/images/installed-world-marker.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>World</string>
   </property>
   <property name="toolTip">
    <string>Show world packages</string>
   </property>
  </action>
  <action name="actionSelectInstalled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/pkgtable/images/installed-marker.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Installed</string>
   </property>
   <property name="toolTip">
    <string>Show installed packages</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>&amp;About Qt ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
